engine/src/main/java/org/destinationsol/drawable/GraphicsElement.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.drawable;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.math.Vector2;
+import org.destinationsol.game.drawables.DrawableLevel;
+
+public class GraphicsElement {
```

Comment:
Javadoc at the class level for everything would be very much appreciated and super useful. There is so much old Terasology code that has no class-level javadoc and over time people forget what the things do. Here we're still at the very beginning and can start out everything right :-)